src/nvim/ex_cmds.h

```diff
@@ -26,6 +28,37 @@ typedef struct {
   list_T *additional_elements;  ///< Additional data left from ShaDa file.
 } SubReplacementString;
 
+
+// Defs for inc_sub functionality
+
+// List of matches in a line
+#define _noop(x)
+KLIST_INIT(colnr_T, colnr_T, _noop)
```

Comment:
I think we should use kvec here (and below) which is more efficient for an append-only sequence.